admin/app/components/organization-information-section.js

```diff
@@ -1,8 +1,13 @@
 import { action } from '@ember/object';
+import ENV from 'pix-admin/config/environment';
 import Component from '@glimmer/component';
 
 export default class OrganizationInformationSection extends Component {
 
+  get externalURL() {
+    return ENV.APP.METABASE_ORGANIZATION_DETAIL_DASHBOARD_URL + this.args.organization.id;
+  }
+
```

Comment:
Proposition pour ne pas générer d'URL si la variable d'environnement n'est pas configurée :
```suggestion
  get externalDashboardURL() {
    const url = ENV.APP.ORGANIZATION_DASHBOARD_URL;
    return url && (url + this.args.organization.id);
  }
```

Reply:
Et peut-être rajouter un test là dessus

Reply:
Done !

Reply:
Corrigé !